NebulaHost/PacketProcessors/GameHistory/GameHistoryRemoveTechProcessor.cs

```diff
@@ -3,6 +3,8 @@
 using NebulaModel.Packets.GameHistory;
 using NebulaModel.Packets.Processors;
 using NebulaWorld.GameDataHistory;
+using NebulaModel.Logger;
```

Comment:
Logger is not used in this class, right?

Reply:
Removed